features/permanent_slots.feature

```diff
@@ -23,15 +25,21 @@ Feature: permanent slots
     Given postgres1 has a logical replication slot named test_logical with the test_decoding plugin after 10 seconds
     Then Logical slot test_logical is in sync between postgres0 and postgres1 after 10 seconds
     And Logical slot test_logical is in sync between postgres0 and postgres2 after 10 seconds
+    And Logical slot test_logical is in sync between postgres0 and postgres3 after 10 seconds
     And postgres1 has a physical replication slot named test_physical after 2 seconds
     And postgres2 has a physical replication slot named test_physical after 2 seconds
+    And postgres3 has a physical replication slot named test_physical after 2 seconds
 
   @slot-advance
   Scenario: check permanent physical slots that match with member names
-    Given postgres1 has a physical replication slot named postgres0 after 2 seconds
+    Given postgres0 has a physical replication slot named postgres3 after 2 seconds
```

Comment:
Maybe we could check that `postgres0` has `postgres1` too?

Reply:
We are already doing that here: https://github.com/zalando/patroni/pull/2891/files#diff-6dffdf09b22cfc35032e5209953e42980b37c1b696a53598d73f61a1fba1dd1aR13